language/move-lang/src/cfgir/locals/mod.rs

```diff
@@ -294,14 +299,15 @@ fn use_local(context: &mut Context, loc: &Loc, local: &Var) {
                 DisplayVar::Orig(s) => s,
             };
             let msg = format!(
-                "The local {} not have a value due to this position. The local must be assigned a \
-                 value before being used",
+                "The variable {} not have a value due to this position. The local must be \
```

Comment:
Since the "local" in the first sentence is changed to "variable," it might be clearer if the "local" in the second sentence were also changed to "variable."